starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,10 @@
+
+    <h3>List of all Drones in the database:</h3>
+    <a href="../drones/new">Add a new drone</a>
```

Comment:
To navigate to some page we actually are pointing where to go and that route will be visible in the url. That being said we don't need these 2 dots, but instead:
```
          <a href="/drones/new"> Add a new drone </a>
```